docs/odh/grafana/add_grafana_dashboard.md

```diff
@@ -0,0 +1,37 @@
+# Add a grafana dashboard to OperateFirst Grafana
+
+To be able to include your dashboards in Grafana, you will need to add a `GrafanaDashboard` custom resource. This custom resource is defined as a yaml file that can remotely reference the dashboard contents from a specified URL and falls back to embedded json if the URL can not be resolved.
+
+An example `grafana-dashboard-example.yaml` template is shown below:
+
+```yaml
+---
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: grafana-dashboard-from-url
+  labels:
+    app: grafana
+spec:
+  name: grafana-dashboard-from-url.json
+  url: https://raw.githubusercontent.com/integr8ly/grafana-operator/master/deploy/examples/remote/grafana-dashboard.json
+  json:
```

Comment:
If both the `url` and `json` properties are specified, the `json` value is used only if the `url` is unresolvable (it's a fallback value). I think it's not necessary to keep it here, I would rather see the operator fail to load the dashboard (so we can track it and alert on it) than fallback to an "empty" one, wdyt?

reference:
- https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md
- https://github.com/integr8ly/grafana-operator/issues/45#issuecomment-529467580

Reply:
@tumido good poin! I think it would be nice if we can alert on it as well, will retain only the `url` field then